### ISSUE TITLE
documentation for a11y options

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#372]
These options have identical documentation in Sim.js:

```js
      // Whether accessibility features are enabled or not.
      accessibility: phet.chipper.queryParameters.accessibility,

      // Whether accessibility features are enabled or not.
      virtualCursor: phet.chipper.queryParameters.virtualCursor,
```

Similarly in initialize-globals.js, for the associated query parameters.

@jessegreenberg Can you please clarify the documentation? How do these options differ?

**Comment by jessegreenberg** [comment]
Actually, I think the virtualCursor option can be removed.  It was used to enable prototype behavior for a virtual cursor that behaved like virtual cursors used in prominent screen readers and I should have documented it as such.

It was a nice prototype, and helped jumpstart some new accessibility features but there is no intention of using this option anymore so there is no point in maintaining it.  The option and associated code should be removed from scenery too.

**Comment by jessegreenberg** [comment]
Removed in commit above and I added a bit more documentation to the accessibility option.  Closing.

**Comment by pixelzoom** [comment]
Reopening.  Also need to be removed from initialize-globals.js. I'll handle that.

**Comment by jessegreenberg** [comment]
Thanks!

**Comment by pixelzoom** [comment]
`virtualCursor` also appears in screen-reader.html:

> <iframe id="sim" src="balloons-and-static-electricity_en.html?accessibility&virtualCursor" width="570"

Remove it?

**Comment by jessegreenberg** [comment]
Yes, screen-reader.html can be removed from balloons-and-static-electricity.

**Comment by jessegreenberg** [comment]
I deleted it.
